### DISCUSSION TITLE
Add some contributor/popularity stats next to repo names?

### DISCUSSION THREAD

**Discussion by alexbozhenko** [avelino/awesome-go#4991] [Ideas]
Hi.
Thanks for creating this _awesome_ project.

Would it be possible to add some stats next to the repo name, that would help users decide which projects are worth looking at.
E.g. when I look at the long list at https://github.com/avelino/awesome-go#json and want to check out what is out there, I had to click on every link.
But I know for sure I am not interested in projects with last contribution >N years ago, and only one contributor (e.g. https://github.com/ricardolonga/jsongo)

This is related to https://github.com/avelino/awesome-go/discussions/4871, 
but even if automatic removal is implemented, 
It still would be nice to have some stats at a glance...

Github itself has search interface shows number of stars and last update time.
So if we do, roughly, something like this to build a search query
```
~/code/awesome-go$ grep -A42 '## JSON' README.md   | grep 'https://github.com' | awk -F')' '{print $1}' | awk -F'https://github.com/' '{print "repo:"$2}'  | paste -d' ' -s
repo:spyzhov/ajson repo:simonnilsson/ask repo:cocoonspace/dynjson repo:lucassscaravelli/ej repo:vtopc/epoch repo:valyala/fastjson repo:skanehira/gjo repo:tidwall/gjson repo:ddymko/go-jsonerror repo:nicklaw5/go-respond repo:limiu82214/gojmapr repo:elgs/gojq repo:ChimeraCoder/gojson repo:nikolaydubina/htmljson repo:yazgazan/jaydiff repo:wI2L/jettison repo:romshark/jscan repo:m-zajac/json2go repo:AmuzaTkts/jsonapi-errors repo:neilotoole/jsoncolor repo:wI2L/jsondiff repo:miolini/jsonf repo:ricardolonga/jsongo repo:RichardKnop/jsonhal repo:abusomani/jsonhandlers repo:sinhashubham95/jsonic repo:Andrew-M-C/go.jsonvalue repo:zerosnake0/jzon repo:Qntfy/kazaam repo:mickep76/mapslice-json repo:PerimeterX/marshmallow repo:sanbornm/mp repo:ohler55/ojg repo:dedalqq/omg.jsonparser repo:olvrng/ujson repo:miladibra10/vjson
```
We can make each section title a link to such query maybe?
[github search example](https://github.com/search?q=repo%3Aspyzhov%2Fajson+repo%3Asimonnilsson%2Fask+repo%3Acocoonspace%2Fdynjson+repo%3Alucassscaravelli%2Fej+repo%3Avtopc%2Fepoch+repo%3Avalyala%2Ffastjson+repo%3Askanehira%2Fgjo+repo%3Atidwall%2Fgjson+repo%3Addymko%2Fgo-jsonerror+repo%3Anicklaw5%2Fgo-respond+repo%3Alimiu82214%2Fgojmapr+repo%3Aelgs%2Fgojq+repo%3AChimeraCoder%2Fgojson+repo%3Anikolaydubina%2Fhtmljson+repo%3Ayazgazan%2Fjaydiff+repo%3AwI2L%2Fjettison+repo%3Aromshark%2Fjscan+repo%3Am-zajac%2Fjson2go+repo%3AAmuzaTkts%2Fjsonapi-errors+repo%3Aneilotoole%2Fjsoncolor+repo%3AwI2L%2Fjsondiff+repo%3Amiolini%2Fjsonf+repo%3Aricardolonga%2Fjsongo+repo%3ARichardKnop%2Fjsonhal+repo%3Aabusomani%2Fjsonhandlers+repo%3Asinhashubham95%2Fjsonic+repo%3AAndrew-M-C%2Fgo.jsonvalue+repo%3Azerosnake0%2Fjzon+repo%3AQntfy%2Fkazaam+repo%3Amickep76%2Fmapslice-json+repo%3APerimeterX%2Fmarshmallow+repo%3Asanbornm%2Fmp+repo%3Aohler55%2Fojg+repo%3Adedalqq%2Fomg.jsonparser+repo%3Aolvrng%2Fujson+repo%3Amiladibra10%2Fvjson&type=repositories&s=updated&o=desc)
It also allows to sort the search results:
![image](https://github.com/avelino/awesome-go/assets/5295986/911d3213-3438-413a-afb7-fb00b77b3dda)

Not sure if there is a way to make the search results page to display more details about each repo...

**Comment by alexbozhenko** [comment]
Found 
https://docs.deps.dev/api/v3alpha/

Which actually is linked to from pkg.go.dev
E.g. here
https://pkg.go.dev/github.com/sirupsen/logrus
there is a link on the right column.

**Comment by alexbozhenko** [comment]
Also, found this: https://github.com/Cyanhall/awesome-go

**Comment by avelino** [comment]
I like the idea of having more meta information, but I think it will make it difficult to maintain the readme, we need to make the readme easy to review and especially to maintain.

now on awesome-go.com is a place where we can "automatically" put this information when generating the html

[I recommend reading](https://github.com/avelino/awesome-go/issues/1213)

**Comment by emanuelef** [comment]
@alexbozhenko I've created a project that is fetching daily this list and creates some stats: https://emanuelef.github.io/awesome-go-repo-stats/
Maybe this can be useful to you.
